tests/end_to_end/test_vulnerability_detector/test_vulnerability_detector.py

```diff
@@ -109,6 +109,70 @@ def load_vulnerability_detector_configurations(host_manager):
 
     return configurations
 
+@pytest.fixture(scope='module')
```

Comment:
This fixture will never be used in the test. Fixtures should be included in the desired tests or marked as autouse. Check the fixture documentation https://docs.pytest.org/en/6.2.x/fixture.html

Reply:
Fixed here: https://github.com/wazuh/wazuh-qa/pull/5158/commits/2126cdd2faa6275ab3ab8daa8820b6e889bc0f32